core/frontend/src/RealityDataDisplayStyle.ts

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { ClipVector } from "@itwin/core-geometry";
+
+/** This class provides a container and methods to specify display properties or actions on reality data. */
+export class RealityDataDisplayStyle {
+  public readonly clips: ClipVector | undefined;
+
+  public constructor(jsonProperties: { [key: string]: any }) {
```

Comment:
Make constructor private. Make static methods to instantiate - one from your JSON representation; another accepting an Options interface that would include (for now) a ClipVector.
Define an interface type for `jsonProperties`, probably called `RealityDataDisplayStyleProps`.
Consider renaming this class to reduce confusion with `DisplayStyle`.

Reply:
Any suggestions for renaming the class? I could rename it something like RealityDataDisplaySettings but then RealityDataDisplaySettingsProps seems a bit weird to me... for now this class only contains clips but could potentially contain properties like transparency, color override, etc. That's why it made sense to me to call it RealityDataDisplayStyle.

Reply:
I hate naming things. :) I don't have any good suggestions, so if you don't either, feel free to resolve this comment.

We have this unfortunate sort of dual-worlds thing with reality models: some of them live in the SpatialModels ("attached" reality models), and others live in DisplayStyles ("context" reality models). Both can be used to create a RealityModelTileTree via `createRealityTileTreeReference`, but while they overlap a lot in functionality they overlap very little in APIs.
A context reality model can already have transparency+color overrides (`ContextRealityModelProps.appearanceOverrides`) but can't currently specify a clip (but that seems like it would be useful).
Now we have attached reality models that can specify a clip, but can't (yet) specify appearance overrides.
I am wondering if some consolidation is appropriate here. But probably not as part of this PR.

Reply:
I've made some changes to the API: 0d1f1bd